webapp/src/language/en.json

```diff
@@ -86,7 +86,63 @@
     "community": "Community",
     "changelog": "Changelog",
     "github": "Github Repository",
-    "telegram": "Telegram Channel"
+    "telegram": "Telegram Channel",
+    "lacchain": "LACChain EOSIO Network Dashboard and Infrastructure Monitor",
+    "lacchain/cpu-benchmark>title": "CPU Benchmarks  - LACChain EOSIO Network Dashboard",
+    "lacchain/block-producers>title": "Block Producers - LACChain EOSIO Network Dashboard",
+    "lacchain/rewards-distribution>title": "Rewards Distribution - LACChain EOSIO Network Dashboard",
```

Comment:
@tetogomez there is a specific lacchain language file that should have the specific translations for that network